configs/_vue-generic.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
   extends: [
+    'plugin:vue/base',
```

Comment:
Looks like this is included in the vue/recommended and vue/vue3-recommended configs. So we probably don't need to include it here again (https://eslint.vuejs.org/user-guide/#bundle-configurations)